worldengine/generation.py

```diff
@@ -105,12 +105,20 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
     """
     e = world.elevation['data']
     ocean = fill_ocean(e, ocean_level)
-    hl = find_threshold_f(e, 0.10)  # the highest 10% of all (!) land are declared hills
-    ml = find_threshold_f(e, 0.03)  # the highest 3% are declared mountains
+
+    # values chosen to emulate the exponential behavior seen here:
+    # http://www.ngdc.noaa.gov/mgg/global/etopo1_surface_histogram.html
+    pl = find_threshold_f(e, 0.80, ocean=ocean)  # the highest x% of land are declared plains; save some for beaches (?)
+    hl = find_threshold_f(e, 0.40, ocean=ocean)  # the highest x% are declared hills
+    ml = find_threshold_f(e, 0.20, ocean=ocean)  # the highest x% are declared low mountains
+    mml = find_threshold_f(e, 0.10, ocean=ocean)  # the highest x% are declared medium mountains
+    hml = find_threshold_f(e, 0.05, ocean=ocean)  # the highest x% are declared high mountains
```

Comment:
I changed the values a bit, created new plots and want to collect all the important points in a single post:
The former "hills"-level was assigned to the highest 3% of the world, the following 7% (for a total of 10%) was declared plains.

The new values declare the following values, in relation to *total* land (in the code the values are relative to non-submerged land):
plains ~28%, hills 14%, low mountains 7%, medium mountains 3.5% and high mountains 1.8%.

So the former "hills" roughly correspond to "medium mountains" now. Thus the changes to river-, temperature- and ancient map-generation. The choices in there made sense, but the hill-level was very high.

The following two plots show the old and new height-levels:
![seed_13051_hypsographic](https://cloud.githubusercontent.com/assets/12257447/11299626/977d8df4-8f88-11e5-9a7d-2498b3fe6c12.png)
![seed_13051_hypsographic](https://cloud.githubusercontent.com/assets/12257447/11299632/a161e216-8f88-11e5-9aaf-3165db61f304.png)

The plains are set to 80% since due to possible depressions in the land 100% could end up below sea-level. 80% leaves a little wiggle-room.
Everything else is chosen to follow a rough 2^x shape, as I estimated it from here (supposedly real data): ![hypsographic curve Earth](http://www.ngdc.noaa.gov/mgg/image/fig1_global_histogram.png)

If anybody has more accurate data, please point me to it (other criticism is welcome, too). For now the values seem reasonable and [the results of world-generation](https://github.com/Mindwerks/worldengine-data/pull/17/files) look good to me.